runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/GrpcGetDataStream.java

```diff
@@ -342,12 +419,15 @@ private <ResponseT> ResponseT issueRequest(QueuedRequest request, ParseFn<Respon
         }
       } catch (IOException e) {
         LOG.error("Parsing GetData response failed: ", e);
+        try {
+          BackOffUtils.next(Sleeper.DEFAULT, backoff);
+        } catch (IOException | InterruptedException ie) {
+          Thread.currentThread().interrupt();
```

Comment:
why interrupt on IOException? 

I'm also not sure why `BackOffUtils.next` has a throws IOException, it doesn't look like any implementation throws.